src/OpenTelemetry.Exporter.Jaeger/Implementation/JaegerConversionExtensions.cs

```diff
@@ -49,6 +49,13 @@ public static JaegerSpan ToJaegerSpan(this Span span)
             if (span?.Attributes is IEnumerable<KeyValuePair<string, object>> attributeMap)
             {
                 jaegerTags = attributeMap.Select(a => a.ToJaegerTag()).ToList();
+            }
+
+            // Add any Span resource attributes as Jaeger tags
+            // Resource attributes should follow https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-resource-semantic-conventions.md
+            if (span?.LibraryResource.Attributes is IEnumerable<KeyValuePair<string, object>> resourceAttributeMap)
+            {
+                jaegerTags.AddRange(resourceAttributeMap.Select(a => a.ToJaegerTag()));
```

Comment:
should service.name be treated specially and placed here: https://github.com/jaegertracing/jaeger-idl/blob/340c869f1b8d7029a132fdc65a83d3b63c0b4fa2/thrift/jaeger.thrift#L68 ?

Reply:
Yeah, maybe all the resource attributes should go there instead of normal tags?

Reply:
Actually looking at this more closely, the service name and process tags (which should be resource attributes I think) are already hooked up in the Jaeger exporter options class, so this PR is not appropriate I think.